### PR TITLE
Voting proces and orders display

### PR DESCRIPTION
**Pull request by pattypooh**
----------
Fixed 2 queries to retrieve:
1) The suggested places for a food meeting
2) Votes per user for a specific food meeting

Orders
---------
Changed the orders view, instead of cards, it was added a table view.

### CODE REVIEW

File: src/main/java/org/jala/efeeder/suggestion/GetSuggestionsCommand.java [hunk 1]
```diff
@@ -27,22 +27,37 @@
...
 public class GetSuggestionsCommand implements CommandUnit {
 
-	private static final String SELECT_USERS_BY_MEETING_SQL =
-            "SELECT id, name, last_name FROM user WHERE id in (SELECT id_user FROM food_meeting_participants WHERE id_food_meeting = ?)";
+	
     private static final String SELECT_PLACES_BY_MEETING_SQL =
-            "SELECT id, name, description, phone, direction, image_link FROM places WHERE id IN"
-            + "(SELECT id_place FROM food_meeting_suggestions WHERE id_food_meeting = ?)";
+    		"SELECT p.id, p.name, p.description, p.phone, p.direction, p.image_link "
+    	    		+" FROM places p, food_meeting_suggestions fms"
```
Comment (rayaviri) [inline src/main/java/org/jala/efeeder/suggestion/GetSuggestionsCommand.java:33]: add one space between +"   like the next + "

File: src/main/java/org/jala/efeeder/suggestion/GetSuggestionsCommand.java [hunk 2]
```diff
@@ -27,22 +27,37 @@
...
             "(SELECT id_place FROM efeeder.food_meeting_suggestions WHERE id = suggestionId) AS placeId " +
             "FROM efeeder.votes WHERE id_food_meeting = ? GROUP BY id_suggestion ORDER BY votes DESC LIMIT 0, 2";
+    
+    //20170118 pescalera: The query now retrieves the right voted place for every user at a specific food meeting. 
     private static final String GET_PARTICIPANT_VOTED_PLACE = 
-            "SELECT id_place FROM food_meeting_suggestions WHERE id = " +
-            "(SELECT id_suggestion FROM votes WHERE id_participant = " +
-            "(SELECT id FROM food_meeting_participants WHERE id_user = ? LIMIT 1) LIMIT 1) LIMIT 1; ";
+    		"SELECT fmp.id_user as id_user, fms.id_place as id_place, "
+    		+" u.last_name as last_name, u.name as name"
```
Comment (rayaviri) [inline src/main/java/org/jala/efeeder/suggestion/GetSuggestionsCommand.java:44]: one white space

**Review by rayaviri — APPROVED** [review]
please only change the comments and after that merge your changes

**Comment by pattypooh** [comment]
Done: Fixes for some queries and a new view for orders.

**Pull request merged**